### ISSUE TITLE
It just suddenly stopped working

### ISSUE THREAD

**Issue by Nuuuuudel** [reek/anti-adblock-killer#4619]
![image](https://github.com/reek/anti-adblock-killer/assets/148394433/71849c5f-2625-4857-98fe-5232f7d69831)

any idea what i should do?

**Comment by Planet-Xplorer** [comment]
Yea, it suddenly stopped working for me too. I've been using this in combination with a script on uBlock Origin but now both have stopped working. This is actually the only reason I made a GitHub Account lol.

**Comment by ghost** [comment]
Yes. It suddenly stopped working 7 years ago! https://github.com/reek/anti-adblock-killer/commits/master

**Comment by VeldrinMaven** [comment]
It doesn't work for me either. It worked yesterday. But it's logical since Youtube is changing their scripts (or whatever it's called) frequently. An alternative for me is to download the youtube video with JDownloader and just watch it offline with MPC-HC or any other video player. It's less distracting that way too. Win-Win.

**Comment by ghost** [comment]
@Nuuuuudel @Planet-Xplorer @VeldrinMaven For YouTube Delete ABP / AdBlock => install uBlock Origin / AdGuard (only one)

This project is very old and unmaintained, I created a new one for AdBlock => https://raw.githubusercontent.com/AdCrunchSoftware/AdCrunchFiltering/main/ABP-syntax-antiadblock.txt
